nixos/tests/common/acme/server/generate-certs.nix

```diff
@@ -11,16 +11,13 @@ let
 in mkDerivation {
   name = "test-certs";
   buildInputs = [ minica ];
-  phases = [ "buildPhase" "installPhase" ];
 
-  buildPhase = ''
+  buildCommand = ''
```

Comment:
is this acceptable?

Reply:
I do like to get `phases` deprecated, but I do not like the general approach in this PR. I already did some other PR which fixes some problems nicer.

Most of the remaining packages require external setup file, which can not be easily tested by the CI.
I could provide fixes, but mostly they are untested: https://github.com/NixOS/nixpkgs/pull/213235
Any suggestions on how to proceed with these packages?

Linking the issue to the PR: https://github.com/NixOS/nixpkgs/issues/28910